### ISSUE TITLE
Cache Cleaner

### ISSUE THREAD

**Issue by MarkyMarkDE** [d4rken-org/sdmaid-se#1136]
Is there maybe a way, to kill Apps and Services before cleaning the cache?
We have the accessibility service to do? 
Seems that the Cache of some running apps (activated in SDMaid settings) can't be deleted, but SDMaid didn't report it.

I mean:
Playstore
Play Services 

So it would maybe helpful, if SDMaid was able to kill this apps first and then clean it's caches.
You know how much problems exists with both caches ...

Maybe you can lable this ticket as enhancement request.

**Comment by d4rken** [comment]
The only thing SD Maid could do would be to "force stop" the app before clearing cache.
The issue with that is that a force-stopped will not launch until reboot or manually opened.
Doing this for all apps could lead to unexpected behavior, i.e. the user asking why some apps are not working :thinking:

**Comment by MarkyMarkDE** [comment]
> The only thing SD Maid could do would be to "force stop" the app before clearing cache. The issue with that is that a force-stopped will not launch until reboot or manually opened. Doing this for all apps could lead to unexpected behavior, i.e. the user asking why some apps are not working 🤔

no, no, it would be great if we can do this only with Playstore and the Play Services thing as it seems, that SD-Maid can't delete this caches when both Apps are currently running. You know how much Problems both caches can result (paid but still basic, e.g.) ...
